pkg/nfs/README.md

```diff
@@ -39,20 +39,14 @@ $ sudo ./_output/nfsplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode -v=
 ```
 
 ## Test
-Get ```csc``` tool from https://github.com/thecodeteam/gocsi/tree/master/csc
+Get ```csc``` tool from https://github.com/rexray/gocsi/tree/master/csc
```

Comment:
Can you do a global search/replace in the entire repo? github.com/thecodeteam/gocsi and "csc identity supported-versions" are also referenced in several other READMEs.

Reply:
Good point.  I noticed that too.  I didn't change others because I don't have an environment to test all other drivers, but this change should be safe to make.  Will update the PR.

Reply:
Done.

Reply:
To make things even more confusing, we decided not to use GoCSI mock driver as despite  the fact that it is great, it introduces lots of functions which we do not need and also it might introduce delays in accepting certain changes csi-sanity needs.
You can see under csi-test, mock folder with new mock driver. It is working and there is an effort to make it even more lighter but without sacrificing functionality.
In the documentation we should use local mock driver.

Reply:
The mock driver is a CSI plugin itself, right?  How can you use that to test a real NFS/Cinder/iSCSI CSI plugin?
Here we are using CLIs from GoCSI.  Are there CLIs included already in csi-test?
Maybe this is something we need to discuss more about.

Reply:
Please check hack/e2e-hostpath.sh, it gives example how other plugins could be tested with csi-sanity. If you replace APP=**hostpathplugin** by nfsplugin binary. You should be able to test it (in theory) ;)

Reply:
Awesome!  I'll give that a try.  Thanks!